### PR TITLE
Target Profile259

### LINKED ISSUE #54: Error when using with Xamarin
### Description

I get the following error when i try to install the package in a Xamarin Android project. Is there a way to enable SwaggerProvider for Xamarin/Mono?

image
### Repro steps
1.  Create a Xamarin Android project in Visual Studio
2.  Try to install the package (v0.5.6)

### PR DESCRIPTION
**Pull request by sergey-tihon**
This PR may fix #54 ☺️

TODOs:
- [x] Target FSharp.Core.dll `4.4.0.0` from `NuGet` package and `Profile259`

### CODE REVIEW

File: src/SwaggerProvider/SwaggerProvider.fsproj [hunk 1]
```diff
@@ -10,10 +10,11 @@
     <RootNamespace>SwaggerProvider</RootNamespace>
     <AssemblyName>SwaggerProvider</AssemblyName>
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
-    <TargetFSharpCoreVersion>4.4.0.0</TargetFSharpCoreVersion>
+    <TargetFrameworkProfile>Profile259</TargetFrameworkProfile>
+    <TargetProfile>netcore</TargetProfile>
+    <TargetFSharpCoreVersion>3.1.2.5</TargetFSharpCoreVersion>
```
Comment (dsyme) [inline src/SwaggerProvider/SwaggerProvider.fsproj]: I thought you were targeting profile 259?  (or should the name of the issue change?)

Comment (sergey-tihon) [inline src/SwaggerProvider/SwaggerProvider.fsproj]: Definitly `Profile259`,  I've created PCL project in VS 2017 and manually merge changes in existing SwaggerProvider.fsproj file, probably did something wrong (this is my first PCL proj)

This is what I get from VS2017
```xml
  <PropertyGroup>
    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
    <SchemaVersion>2.0</SchemaVersion>
    <ProjectGuid>0d200fdc-8f8a-43a0-bd11-d30b53eb98f8</ProjectGuid>
    <OutputType>Library</OutputType>
    <RootNamespace>PortableLibrary_Profile259</RootNamespace>
    <AssemblyName>PortableLibrary_Profile259</AssemblyName>
    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
    <TargetFrameworkProfile>Profile259</TargetFrameworkProfile>
    <TargetProfile>netcore</TargetProfile>
    <TargetFSharpCoreVersion>3.259.41.0</TargetFSharpCoreVersion>
    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
    <Name>PortableLibrary_Profile259</Name>
  </PropertyGroup>
```

Comment (sergey-tihon) [inline src/SwaggerProvider/SwaggerProvider.fsproj]: Template in `generator-fsharp` also have all these lines... https://github.com/fsprojects/generator-fsharp/pull/25/files 

Does it incorrect?

Comment (dsyme) [inline src/SwaggerProvider/SwaggerProvider.fsproj]: That's fine, or 
```
 <TargetFSharpCoreVersion>3.259.4.0</TargetFSharpCoreVersion>
```
with FSharp.Core nuget 4.0.0.1

Comment (dsyme) [inline src/SwaggerProvider/SwaggerProvider.fsproj]: Some notes here: http://fsharp.github.io/2015/04/18/fsharp-core-notes.html#fsharpcore-entries-in-project-files

**Comment by sergey-tihon** [comment]
Not going to support Profile259, I hope that we will compile SwaggerProvider to .NET Standard one day using new TP SDK

**Pull request closed without merging**